index.html

```diff
@@ -9,6 +9,10 @@ <h2> Here are the people in the class </h2>
 <li><a href="https://www.vijayp.ca/">Vijay Pandurangan</a> (coinstructor) </li>
 </ul>
 
+<h2> Sophia Wang </h2> 
```

Comment:
I think it would be better to add this in as a list item, i.e. between <li> and </li> between the unordered list (ul) tags above.